pycytominer/cyto_utils/load.py

```diff
@@ -1,10 +1,58 @@
 import csv
 import gzip
+import pathlib
 import numpy as np
 import pandas as pd
+from typing import Union
 
 
-def infer_delim(file):
+def is_path_a_parquet_file(file: Union[str, pathlib.Path]) -> bool:
+    """Checks if the provided file path is a parquet file.
+
+    Identify parquet files by inspecting the file extensions.
+    If the file does not end with `parquet`, this will return False, else True.
+
+    Parameters
+    ----------
+    file : Union[str, pathlib.Path]
+        path to parquet file
+
+    Returns
+    -------
+    bool
+        Returns True if the file path contains `.parquet`, else it will return
+        False
+
+    Raises
+    ------
+    TypeError
+        Raised if a non str object is passed in the `file` parameter
+    FileNotFoundError
+        Raised if the provided path in the `file` does not exist
+    """
+    # type checking
+    if not isinstance(file, (str, pathlib.Path)):
+        raise TypeError(f"file must be a str not {type(file)}")
+
+    # convert str to pathlib.Path object
+    if isinstance(file, str):
+        file = pathlib.Path(file).absolute()
+
+    # if a pathlib.Path object is passed check if it is an absolute path
+    # -- if not, get absolute path
+    if not file.is_absolute():
+        file = file.absolute()
+
+    # checking if the file exists
+    if not file.exists():
+        raise FileNotFoundError(f"{str(file.absolute())} does not exist")
+    # checking if file path is a parquet file
+    if not file.suffix.lower() == ".parquet":
+        return False
+    return True
```

Comment:
```suggestion
    # Check if file path is a parquet file
    if file.suffix.lower() == ".parquet":
        return True
    
    return False
```

Avoiding the double negative makes things only marginally easier to read, and might be preferable.